### PR TITLE
Expand NPC backstory "Prisoner 1"

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary

```SUMMARY: Content "Expands NPC backstory 'prisoner 1'"```

#### Purpose of change

This backstory is currently incomplete.

#### Describe the solution

 I have added most of the written story with the exception of the missions that will go along with it later.

I have also added a number of mostly fun "what were you in for" questions based on what expertises or traits the prisoner might have. Some of these will probably lead to conversations of their own later on. For now I'm just leaving them as stubs. I plan to do a few more prisoner backstories, each with their own set of answers to the question.

#### Describe alternatives you've considered

I don't need the big "what were you in for" segment, but it's fun.

### CODE REVIEW

File: data/json/npcs/Backgrounds/prisoner_1.json [hunk 1]
```diff
@@ -3,7 +3,271 @@
...
+  {
+    "id": "BGSS_PRISONER_1_IN_FOR_OTHER",
+    "type": "talk_topic",
+    "dynamic_line": "You know, I don't really want to say anymore.  It's all behind me, and I'd like to keep it that way.",
+    "responses": [ { "text": "<done_conversation_section>", "topic": "TALK_NONE" }, { "text": "<end_talking>", "topic": "TALK_DONE" } ]
+  },
+  {
+    "id": "BGSS_PRISONER_1_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "Okay, well, I was in the wrong place at the wrong time. There was a big fight, I didn't stay clear of it, and me and a bunch of others got tossed in solitary while a few more landed in the infirmary. Some looked pretty bad, now I kinda wonder if any of them were our first <zombies>.",
```
Comment (ZhilkinSerg) [inline data/json/npcs/Backgrounds/prisoner_1.json]:
```suggestion
    "dynamic_line": "Okay, well, I was in the wrong place at the wrong time.  There was a big fight, I didn't stay clear of it, and me and a bunch of others got tossed in solitary while a few more landed in the infirmary.  Some looked pretty bad, now I kinda wonder if any of them were our first <zombies>.",
```

File: data/json/npcs/Backgrounds/prisoner_1.json [hunk 2]
```diff
@@ -3,7 +3,271 @@
...
+    "responses": [
+      { "text": "How did you get out of lockup?", "topic": "BGSS_PRISONER_1_STORY3" },
+      { "text": "<done_conversation_section>", "topic": "TALK_NONE" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_PRISONER_1_STORY3",
+    "type": "talk_topic",
+    "dynamic_line": "I heard gunshots, even from down in lockup. Didn't hear much screaming or anything. That was my first clue something was up. Food stopped showing up, next. Then, the lights went out. I was down there for maybe hours, maybe days, when finally a flashlight in the bars blinded me. It was a guard. He let me out, filled me in on what was going on. I wanted to think he was crazy, but something in his eyes... I believed him.",
```
Comment (ZhilkinSerg) [inline data/json/npcs/Backgrounds/prisoner_1.json]:
```suggestion
    "dynamic_line": "I heard gunshots, even from down in lockup.  Didn't hear much screaming or anything. That was my first clue something was up. Food stopped showing up, next.  Then, the lights went out.  I was down there for maybe hours, maybe days, when finally a flashlight in the bars blinded me.  It was a guard.  He let me out, filled me in on what was going on.  I wanted to think he was crazy, but something in his eyes... I believed him.",
```

File: data/json/npcs/Backgrounds/prisoner_1.json [hunk 3]
```diff
@@ -3,7 +3,271 @@
...
+    "responses": [
+      { "text": "What did you do from there?", "topic": "BGSS_PRISONER_1_STORY4" },
+      { "text": "<done_conversation_section>", "topic": "TALK_NONE" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_PRISONER_1_STORY4",
+    "type": "talk_topic",
+    "dynamic_line": "We let out the others in solitary. We were stuck in, the guard bots had gone haywire and wouldn't let anyone out, and the rest of the people except this one guard had turned. We spent a few days pulping <zombies> and trying to figure a safe way past the bots. Food was running short. Finally we picked the worst, only plan we could think of: we dragged some storage lockers to the entry hall, used them as shields, and pushed them until we were close enough to take out the bots' sensors with our weapons.",
```
Comment (ZhilkinSerg) [inline data/json/npcs/Backgrounds/prisoner_1.json]:
```suggestion
    "dynamic_line": "We let out the others in solitary.  We were stuck in, the guard bots had gone haywire and wouldn't let anyone out, and the rest of the people except this one guard had turned.  We spent a few days pulping <zombies> and trying to figure a safe way past the bots.  Food was running short.  Finally we picked the worst, only plan we could think of: we dragged some storage lockers to the entry hall, used them as shields, and pushed them until we were close enough to take out the bots' sensors with our weapons.",
```

File: data/json/npcs/Backgrounds/prisoner_1.json [hunk 4]
```diff
@@ -3,7 +3,271 @@
...
+    "responses": [
+      { "text": "Did that actually work?", "topic": "BGSS_PRISONER_1_STORY5" },
+      { "text": "<done_conversation_section>", "topic": "TALK_NONE" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_PRISONER_1_STORY5",
+    "type": "talk_topic",
+    "dynamic_line": "It worked better than I'd imagined, honestly. We thought the bots would shoot the lockers but I guess they mistook us for family. There were six of us and four of them, and four of us made it out.",
```
Comment (ZhilkinSerg) [inline data/json/npcs/Backgrounds/prisoner_1.json]:
```suggestion
    "dynamic_line": "It worked better than I'd imagined, honestly.  We thought the bots would shoot the lockers but I guess they mistook us for family.  There were six of us and four of them, and four of us made it out.",
```

File: data/json/npcs/Backgrounds/prisoner_1.json [hunk 5]
```diff
@@ -3,7 +3,271 @@
...
+    "responses": [
+      { "text": "What happened to the others that made it?", "topic": "BGSS_PRISONER_1_STORY6" },
+      { "text": "<done_conversation_section>", "topic": "TALK_NONE" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_PRISONER_1_STORY6",
+    "type": "talk_topic",
+    "dynamic_line": "The guard took off on his own. Didn't trust us, and I don't blame him. The other two wanted to set up a bandit gig. Didn't sit right with me, so I split on pretty good terms. I ran into the guard a couple more times. Thought of seeing if he'd travel with me, but I dunno. I don't think he'd take the offer, I'll always be a con to him. If you want to try, I can tell you where I saw him last. Wasn't long before I met you, and he had a good thing going, might still be there.",
```
Comment (ZhilkinSerg) [inline data/json/npcs/Backgrounds/prisoner_1.json]:
```suggestion
    "dynamic_line": "The guard took off on his own.  Didn't trust us, and I don't blame him.  The other two wanted to set up a bandit gig.  Didn't sit right with me, so I split on pretty good terms.  I ran into the guard a couple more times.  Thought of seeing if he'd travel with me, but I dunno.  I don't think he'd take the offer, I'll always be a con to him.  If you want to try, I can tell you where I saw him last.  Wasn't long before I met you, and he had a good thing going, might still be there.",
```

Comment (I-am-Erk) [reply data/json/npcs/Backgrounds/prisoner_1.json]: Dang, forgot to run another search and replace after adding the second set of stories. Thanks.

**Pull request merged**